### ISSUE TITLE
"Missing mandatory column family"

### ISSUE THREAD

**Issue by mars** [Mashape/kong#999]
When starting Kong with an empty Cassandra keyspace, this error is returned:

```
Missing mandatory column family "schema_migrations" in configured keyspace. Please consider running "kong migrations reset" to fix this.
```

**Why doesn't Kong automatically reset migrations when the keyspace is empty?**

If the caution here is to avoid destroying existing data, then could Kong check if any of its core schema already exist, like `apis` or `consumers`, and otherwise proceed with automatic migrations reset.

**Comment by thefosk** [comment]
I can replicate this on Cassandra 3.0, which is not supported by Kong. Maybe it's a driver problem with Cassandra 3.0.

**Comment by thefosk** [comment]
@mars what version of Cassandra were you using when you experienced this problem?

**Comment by mars** [comment]
I encountered this error w/ Cassandra version 2.1.12.

**Comment by LucaGallinari** [comment]
Same for me. I am using Cassandra 2.2.5 and kong with docker.

Found this error while querying the "schema_migrations":
> cqlsh:kong> select * from schema_migrations;
Traceback (most recent call last):
  File "/usr/bin/cqlsh.py", line 1211, in perform_simple_statement
    result = future.result()
  File "/usr/share/cassandra/lib/cassandra-driver-internal-only-3.0.0-6af642d.zip/cassandra-driver-3.0.0-6af642d/cassandra/cluster.py", line 3122, in result
    raise self._final_exception
ReadFailure: code=1300 [Replica(s) failed to execute read] message="Operation failed - received 0 responses and 1 failures" info={'failures': 1, 'received_responses': 0, 'required_responses': 1, 'consistency': 'ONE'}

One big problem is that I cannot launch the "migration reset" command inside the kong container while it is stopped. And if I try to start it, it will stop immediately and display the error of this issue.

**Comment by allisthere2love** [comment]
@LucaGallinari , if your problem is just to run "migration reset", you can run you kong docker container with the cassandra contact_point var but using "--entrypoint bash" and "-ti". That way you'll have a terminal and you can run "kong migrations reset". Or you can just drop keyspace kong in your cassandra.

About this missing column_family error, are you running cassandra in default port? I had this problem when running 2 cassandras in same host, one with default ports and the other with custom ports. This problem of mine is fixed in kong 0.8

**Comment by thibaultcha** [comment]
Feel free to reopen if still happening as of 0.9.
